tests/io/test_path.py

```diff
@@ -406,7 +406,7 @@ def test_asset(self):
         attach("s3", fs=FakeRemoteFileSystem())
 
         p = "s3"
-        f = "/tmp/foo"
+        f = "tmp/foo"
```

Comment:
Why the change here?

Reply:
it should be `s3://../..` instead of `s3:///../..`

Reply:
Ah I see. We should do this instead to make it more obvious

```suggestion
        f = "bucket/object"
```

Reply:
sounds like a great idea!